pkg/forth/forth.go

```diff
@@ -362,6 +363,13 @@ func roundup(f Forth) {
 	f.Push(v)
 }
 
+func fmtprintf(f Forth) {
```

Comment:
I'm not familiar with Forth, but if I just want to print string say `hello`, in need to send two operands anyway? like `"hello _ printf"`